src/components/page-list/page-list.jsx

```diff
@@ -19,13 +19,12 @@ import './page-list.css';
  */
 export default class PageList extends React.Component {
   render () {
-    if (!this.props.pages) {
-      return <Loading />;
-    }
-
     let results;
 
-    if (this.props.pages instanceof Error) {
+    if (!this.props.pages){
```

Comment:
Tiny issue: can you add a space after the closing parenthesis?

Reply:
Yea no problem!